### PR TITLE
Adding --theme-top-bar-search-*

### PR DESCRIPTION
**Pull request by Link2Twenty**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
Adding both --theme-top-bar-search-color and --theme-top-bar-search-background to change the colour and background colour of the search box.

## Related Tickets & Documents
#1104 and #1377

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/assets/stylesheets/top-bar.scss [hunk 1]
```diff
@@ -311,6 +311,13 @@
     padding: 8px;
     font-size: 0.9em;
     background: #e8e7e7;
```
Comment (maestromac) [inline app/assets/stylesheets/top-bar.scss]:
```suggestion
```
I believe you intended to replace this line with the 1 line below.

File: app/assets/stylesheets/top-bar.scss [hunk 2]
```diff
@@ -311,6 +311,13 @@
...
+    background: var(--theme-top-bar-search-background, #e8e7e7);
+    color: var(--theme-top-bar-search-color, $black);
     -webkit-appearance: none;
+    &::placeholder {
+      opacity: 0.5;
+      color: $black;
```
Comment (maestromac) [inline app/assets/stylesheets/top-bar.scss]:
```suggestion
```
Meant to erase this too?

File: app/assets/stylesheets/top-bar.scss (hunk 1 above)
Comment (Link2Twenty) [reply app/assets/stylesheets/top-bar.scss]: That is what I did with the earlier ones but I realised that broke things for IE, leaving that line in means IE will still get the original colours

Comment (maestromac) [reply app/assets/stylesheets/top-bar.scss]: Ah I see. Please disregard that then.

Comment (Link2Twenty) [reply app/assets/stylesheets/top-bar.scss]: Unless we don't care about IE, I doubt we get much traffic from IE?

Comment (maestromac) [reply app/assets/stylesheets/top-bar.scss]: @benhalpern , what's your thought on this?

Comment (benhalpern) [reply app/assets/stylesheets/top-bar.scss]: I'm forming the habit of answering questions in the form of documentation updates.

I didn't realize our public docs didn't have browser support listed anywhere.

So... we do not officially support any version of IE.

https://github.com/thepracticaldev/dev.to/pull/1378/files

Comment (Link2Twenty) [reply app/assets/stylesheets/top-bar.scss]: In the vain of not going "out of our way to break anything" should we leave in the initial colours? I'll have to go back and add them to the other 2 commits, bit that's not too much work.

Comment (Link2Twenty) [reply app/assets/stylesheets/top-bar.scss]: In the vain of not going "out of our way to break anything" should we leave in the initial colours? I'll have to go back and add them to the other 2 commits, bit that's not too much work.

**Pull request merged**